src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_Border.cs

```diff
@@ -102,5 +109,224 @@ public async Task When_Border_Centered_With_Margin_Inside_Wide_Rectangle()
+			var bottomTarget = SUT.Get_BottomTarget();
+			var centerTarget = SUT.Get_CenterTarget();
+
+			foreach (var expected in expectedColors)
+			{
+				Thickness t = new Thickness(expected.Thicknesses[0], expected.Thicknesses[1], expected.Thicknesses[2], expected.Thicknesses[3]);
+				SUT.Set_MyBorderThickness(t);
+
+				WindowHelper.WindowContent = SUT;
+				WindowHelper.WaitForLoaded(SUT);
+				var renderer = new RenderTargetBitmap();
+				await WindowHelper.WaitForIdle();
+				await renderer.RenderAsync(SUT);
+				var result = await RawBitmap.From(renderer, SUT);
+				WindowHelper.WaitForIdle();
+
+				ImageAssert.HasColorAtChild(result, leftTarget, (float)leftTarget.Width / 2,
+					(float)leftTarget.Height / 2, expected.Colors[0]);
+				ImageAssert.HasColorAtChild(result, topTarget, (float)topTarget.Width / 2,
+					(float)topTarget.Height / 2, expected.Colors[1]);
+				ImageAssert.HasColorAtChild(result, rightTarget, (float)rightTarget.Width / 2,
+					(float)rightTarget.Height / 2, expected.Colors[2]);
+				ImageAssert.HasColorAtChild(result, bottomTarget, (float)bottomTarget.Width / 2,
+					(float)bottomTarget.Height / 2, expected.Colors[3]);
+				ImageAssert.HasColorAtChild(result, centerTarget, (float)centerTarget.Width / 2,
+					(float)leftTarget.Height / 2, lightBlue);
+			}
+		}
+
+		[TestMethod]
+		public async Task When_Border_CornerRadius_Clipping()
+		{
+			const string red = "#FFFF0000";
+
+			if (!ApiInformation.IsTypePresent("Windows.UI.Xaml.Media.Imaging.RenderTargetBitmap"))
+			{
+				Assert.Inconclusive(); // System.NotImplementedException: RenderTargetBitmap is not supported on this platform.;
+			}
+			var SUT = new Border_CornerRadius_Clipping();
+
+			var topLeftTarget = SUT.Get_TopLeftTarget();
+			var topRightTarget = SUT.Get_TopRightTarget();
+			var bottomLeftTarget = SUT.Get_BottomLeftTarget();
+			var bottomRightTarget = SUT.Get_BottomRightTarget();
+
+			WindowHelper.WindowContent = SUT;
+			WindowHelper.WaitForLoaded(SUT);
+			var renderer = new RenderTargetBitmap();
+			await WindowHelper.WaitForIdle();
+			await renderer.RenderAsync(SUT);
+			var result = await RawBitmap.From(renderer, SUT);
+			WindowHelper.WaitForIdle();
+
+			ImageAssert.HasColorAtChild(result, topLeftTarget, (float)topLeftTarget.Width / 2,
+					(float)topLeftTarget.Height / 2, red);
+			ImageAssert.HasColorAtChild(result, topRightTarget, (float)topRightTarget.Width / 2,
+					(float)topRightTarget.Height / 2, red);
+			ImageAssert.HasColorAtChild(result, bottomLeftTarget, (float)bottomLeftTarget.Width / 2,
+					(float)bottomLeftTarget.Height / 2, red);
+			ImageAssert.HasColorAtChild(result, bottomRightTarget, (float)bottomRightTarget.Width / 2,
+					(float)bottomRightTarget.Height / 2, red);
+		}
+
+		[TestMethod]
+		public async Task When_Border_LinearGradient()
+		{
+			#if __IOS__
+				Assert.Inconclusive(); // iOS not working currently. https://github.com/unoplatform/uno/issues/6749
+			#endif
```

Comment:
preprocessor directives don't affect indentation
```suggestion
#if __IOS__
			Assert.Inconclusive(); // iOS not working currently. https://github.com/unoplatform/uno/issues/6749
#endif
```